src/patternfly/upgrade-examples/AlertUpgradeExamples/examples/alert-upgrade-example-pf3-notification.hbs

```diff
@@ -24,8 +24,8 @@
   </div>
   <br>
   <div class="toast-pf alert alert-info">
-    <div class="dropdown pull-right dropdown-kebab-pf">
-      <button class="btn btn-link dropdown-toggle" type="button" id="dropdownKebabRight" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+    <div class="dropdown pull-right dropdown-kebab-pf" aria-label="dropdown toggle">
```

Comment:
`aria-label` should be removed here. 

A `<div>` generally doesn't support `aria-label` unless it has an explicitly assigned role that does support it (usually roles that are interactive elements like buttons or landmark roles).

Reply:
`aria-label` removed.